source/features/archive-forks-link.tsx

```diff
@@ -23,6 +21,10 @@ void features.add(import.meta.url, {
 	include: [
 		pageDetect.hasRepoHeader,
 	],
+	asLongAs: [
+		pageDetect.isPublicRepo,
+		pageDetect.isArchivedRepo,
```

Comment:
isArchivedRepo can't be used this way, that's why it was inlined. And also it can't have domReady:true because it's way up in the header. 

isPublicRepo can stay here though, it's meta tag based.